docs/migrations/support-slashed-denoms.md

```diff
@@ -0,0 +1,113 @@
+# Migrating from Unsupported Base Denom Slashes to Supporting Slashed BaseDenoms
+
+This document is intended to highlight significant changes which may require more information than presented in the CHANGELOG.
+Any changes that must be done by a user of ibc-go should be documented here.
+
+There are four sections based on the four potential user groups of this document:
+- Chains
+- IBC Apps
+- Relayers
+- IBC Light Clients
+
+**Note:** ibc-go supports golang semantic versioning and therefore all imports must be updated to bump the version number on major releases.
+```go
+github.com/cosmos/ibc-go/v3 -> github.com/cosmos/ibc-go/v4
+```
+
+This document is necessary when chains are upgrading from a version that does not support slashed base denoms (e.g. v3.0.0) to a version that does (e.g. v3.1.0).
+
+If a chain receives a slashed denom before it upgrades to supporting it, the receive may pass however the trace information will be incorrect.
+
+E.g. If a base denom of `testcoin/testcoin/testcoin` is sent to a chain that does not support slashes in the base denom; the receive will be successful. However, the trace information stored on the receiving chain will be: `Trace: "testcoin/testcoin", BaseDenom: "testcoin"`
+
+This incorrect trace information must be corrected when the chain does upgrade to fully supporting slashed denominations.
+
+To do so, chain binaries should include a migration script that will run when the chain upgrades from not supporting slashed base denominations to supporting slashed base denominations.
+
+## Chains
+
+### Transfer
+
+The transfer module will now support slashes in base denoms, so we must iterate over current traces to check if any of them are incorrectly formed and correct the trace information.
+
+### Upgrade Propsoal
+
+```go
+app.UpgradeKeeper.SetUpgradeHandler("v3.1.0",
```

Comment:
I think it's important to mention that this version is the one from the chain that is importing IBC-go